packages/components/organisms/f-checkout/src/components/Accordion.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <div>
+        <button
+            :id="id + '-accordion-header'"
+            :aria-controls="id + '-header'"
+            :aria-expanded="isExpanded"
+            :class="$style['c-checkout-accordion-header']"
+            :data-test-id="id + '-accordion-header'"
+            @click="toggle">
+            {{ title }}
+            <chevron-icon
+                :class="[
+                    $style['c-checkout-accordion-header-arrow'], {
+                        [$style['c-checkout-accordion-header-arrow--expanded']]: isExpanded
+                    }]" />
+        </button>
+        <section
+            :id="id + '-accordion-section'"
+            :aria-labelledby="id + '-section'"
+            :class="[
+                $style['c-checkout-accordion-section'], {
+                    [$style['c-checkout-accordion-section--hidden']]: !isExpanded
+                }]"
+        >
+            <slot />
+        </section>
+    </div>
+</template>
+
+<script>
+import { ChevronIcon } from '@justeat/f-vue-icons';
+
+export default {
+    name: 'Accordion',
+
+    components: {
+        ChevronIcon
+    },
+
+    props: {
+        id: { type: String, required: true },
+        title: { type: String, required: true }
+    },
+
+    data () {
+        return {
+            isExpanded: false
+        };
+    },
+
+    methods: {
+        toggle (e) {
+            e.preventDefault();
+            this.isExpanded = !this.isExpanded;
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+    $icon-size: 16px;
+    $header-colour: #2a3846;
+
+    .c-checkout-accordion-header {
+        background: transparent;
+        padding: spacing(x2) 0;
+        margin-top: spacing(x2);
+        width: 100%;
+        text-decoration: none;
+        text-align: left;
+        outline: none;
+        border: none;
+        border-bottom: 1px solid #c5ccd3;
+        display: flex;
+        color: $header-colour;
+        justify-content: space-between;
+        cursor: pointer;
+        font-size: 1rem;
+        font-weight: 600;
```

Comment:
`$font-weight-bold `

Reply:
I was hoping someone would point me to the correct SCSS variable names for these. 😅

Reply:
😄 They're all here I think https://github.com/justeat/pie-design-tokens/blob/master/pie-design-tokens.jsonc

Reply:
😄 They're all here I think https://github.com/justeat/fozzie/blob/master/src/scss/settings/_variables.scss#L57 so you should be able to do the font-size one too